src/NavBar.js

```diff
@@ -78,11 +78,11 @@ class NavBar extends React.Component {
     ];
 
     return (
-      <View style={navStyles}>
+      <SafeAreaView style={navStyles}>
```

Comment:
Incorrect use of SafeAreaView for NavBar.
SafeAreaView is used as layout/container for the whole App